app/views/admin/AnalyticsView.coffee

```diff
@@ -16,20 +16,15 @@ module.exports = class AnalyticsView extends RootView
   lineColors: ['red', 'blue', 'green', 'purple', 'goldenrod', 'brown', 'darkcyan']
   minSchoolCount: 20
 
-  constructor: (options) ->
-    super options
+  initialize: ->
     @loadData()
-
-  getRenderData: ->
-    context = super()
-    context.activeClasses = @activeClasses ? []
-    context.activeClassGroups = @activeClassGroups ? {}
-    context.activeUsers = @activeUsers ? []
-    context.revenue = @revenue ? []
-    context.revenueGroups = @revenueGroups ? {}
-    context.dayEnrollmentsMap = @dayEnrollmentsMap ? {}
-    context.enrollmentDays = @enrollmentDays ? []
-    context
+    @activeClasses = @activeClasses ? []
+    @activeClassGroups = @activeClassGroups ? {}
+    @activeUsers = @activeUsers ? []
+    @revenue = @revenue ? []
+    @revenueGroups = @revenueGroups ? {}
+    @dayEnrollmentsMap = @dayEnrollmentsMap ? {}
+    @enrollmentDays = @enrollmentDays ? []
```

Comment:
While this works, someone looking at this would wonder why properties might be set to themselves. For reading clarity, the lines can just be
```coffeescript
@activeClasses = []
@activeClassGroups = {}
# etc
```